src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -572,6 +567,14 @@ fn run(mut args: Args) -> Result<(), anyhow::Error> {
             build_info: BUILD_INFO,
         }))?;
 
+    let span = tracing::info_span!("environmentd::run");
+    let _enter = span.enter();
```

Comment:
```suggestion
    let span = tracing::info_span!("environmentd::run").entered();
```

if you want it cleaner! Also, note that this will stay opened, so we await any futures below here, it may be inaccurate. looks like we just `block_on` tho, so its probably not too bad

Reply:
I think all the awaiting happens elsewhere. Thanks for the suggestion, done.